docs/architecture/grpc-for-wcf-developers/why-grpc.md

```diff
@@ -1,11 +1,41 @@
 ---
 title: Why gRPC is recommended for WCF developers - gRPC for WCF Developers
-description: TO BE WRITTEN
+description: A discussion of why gRPC is a good fit for WCF developers looking to migrate to modern architectures and platforms
 author: markrendle
 ms.date: 09/02/2019
 ---
 
 # Why gRPC is recommended for WCF developers
 
+Before diving deeply into the language and techniques of gRPC it is worth pulling together some ideas about why, given there are alternatives available, gRPC is the right solution for WCF developers who want to migrate to .NetCore.
+
+## Similarity to WCF
+
+Although its implementation and approach is different, the actual experience of developing and consuming services with gRPC should be very intuitive for WCF developers. The underlying goal of making it possible to code as though the client and server were on the same platform, without needing to worry about networking, is the same. Both platforms share the principle of declaring and then implementing an interface, even though the process for declaring that interface is different. And as you will see in Chapter 5, the different types of RPC calls supported by gRPC map very well to the different bindings available to WCF services.
+
+## Benefits of gRPC
+
+Additional reasons why gRPC stands above other solutions are:
+
+### Performance
+
+As already discussed, using HTTP/2 rather than HTTP/1.1 removes the requirement for human-readability code and instead use the smaller faster binary protocol. This is more efficient for computers to parse. HTTP/2 also supports multiplexing requests over a single connection enabling responses to be sent as soon as they are ready without the need to wait in a queue.  Fewer resources are needed when using gRPC which makes it a good solution to use for mobile devices and over slower networks. [Mark to check]
+
+### Interoperability
+
+There are gRPC tools and libraries for all major programming languages and platforms, including .NET, Java, Python, Go, C++, Node.js, Swift, Dart, Ruby, and PHP. Thanks to the Protocol Buffers binary wire format and the efficient code generation for each platform, developers can achieve performance better than WCF's NetTCP bindings while still enjoying full cross-platform support.
+
+### Simplicity and productivity
```

Comment:
I recall that the docs team doesn't like the term simplicity - it is broad and can be meaningless. You should be more specific // @mairaw

I described gRPC as being prescriptive instead here - https://docs.microsoft.com/en-us/aspnet/core/grpc/comparison?view=aspnetcore-3.0#strict-specification

Reply:
https://microsoft-ce-csi.acrolinx.cloud/htmldata/en/rules/help/avoid_simply.html
**Avoid "simple/simply"**
It's generally a good idea to avoid describing instructions with simple or simply as the reader might not find it so simple.